### PR TITLE
feat(jupyterlab): new app

### PR DESCRIPTION
**Pull request by greut**
Something is missing on the dbus side.

https://github.com/mviereck/x11docker/blob/master/x11docker#L7351-L7362

### CODE REVIEW

**Comment by greut** [comment]
@floriansipp this is what I get.

```
/docker-entrypoint.d/2-run-app.sh: line 22: [: ==: unary operator expected
Running  as chorus on GPU...
[77:0820/115143.620334:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[77:0820/115144.933770:ERROR:ozone_platform_x11.cc(240)] Missing X server or $DISPLAY
[77:0820/115144.933784:ERROR:env.cc(255)] The platform failed to initialize.  Exiting.
Segmentation fault (core dumped)
 exited.
/docker-entrypoint.sh: Configuration complete; ready for start up
```

**Comment by greut** [comment]
It's starting but don't click anywhere 😜 

Capture d’écran 2024-09-06 à 15 09 52

File: apps/jupyterlab/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        scipy \
+        seaborn \
+        spacy \
+        streamlit \
+        tqdm && \
+    apt-get remove -y --purge \
+        curl && \
+    apt-get autoremove -y --purge
+
+ENV APP_CMD_PREFIX="export PATH=${CONDA_DIR}/bin:${PATH}"
```
Comment (greut) [inline apps/jupyterlab/Dockerfile:88]: the content of this is evaluated at build time.

**Comment by greut** [comment]
Capture d’écran 2024-10-30 à 11 25 00

**Review by floriansipp — APPROVED** [review]

**Pull request merged**